core/src/main/java/com/datastax/oss/driver/api/core/session/Session.java

```diff
@@ -51,15 +52,14 @@
 public interface Session extends AsyncAutoCloseable {
 
   /**
-   * The current version of the core driver (in other words, the version of the {@code
-   * com.datastax.oss:java-driver-core} artifact).
+   * Returns information about the driver, such as its current version number.
    *
    * <p>This is intended for products that wrap or extend the driver, as a way to check
    * compatibility if end-users override the driver version in their application.
    */
-  static String getCoreDriverVersion() {
-    // Note: getBundle caches its result
-    return ResourceBundle.getBundle("com.datastax.oss.driver.Driver").getString("driver.version");
+  default DriverInfo getDriverInfo() {
+    return DefaultDriverInfo.buildFromResource(
+        getClass().getResource("/com/datastax/oss/driver/Driver.properties"));
```

Comment:
👎 on making this an instance method.
There is some value in being able to test the driver version before creating a session. I could imagine some 3rd-party tool needing that.

Reply:
Fair enough, will revert to static. But in this case I think a more explicit method name is better, e.g. `getOssDriverInfo()`.